tools/RAiDER/statsPlot.py

```diff
@@ -710,16 +713,23 @@ def __call__(self, gridarr, plottype, workdir='./', drawgridlines=False, colorba
             if cbounds is None:
                 cbounds = [np.nanpercentile(gridarr, self.colorpercentile[0]), np.nanpercentile(
                     gridarr, self.colorpercentile[1])]
+                # if upper/lower bounds identical, overwrite lower bound as 75% of upper bound to avoid plotting ValueError
+                if cbounds[0]==cbounds[1]:
```

Comment:
Use a linter and/or autoformatter
```suggestion
                if cbounds[0] == cbounds[1]:
```